### PR TITLE
Assg 4 awsirkis

### PR DESCRIPTION
**Pull request by awsirkis**

### CODE REVIEW

File: lesson04/assignment/index.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// JavaScript source code
+var images = ["vsd.jpg", "fdsdb.jpg", "MV5BMjAzNjYwNzkxMl5BMl5BanBnXkFtZTgwMTU1OTk4MDE@._V1_UY317_CR16,0,214,317_AL_.jpg", "benjamin-parker-OhKElOkQ3R-unsplash-cropped.jpg"];
+var i = 0;
+var p = "images/";
```
Comment (brycebenson) [inline lesson04/assignment/index.js]: This path isn't lining up with the folder structure.

File: lesson04/assignment/index.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+// JavaScript source code
+var images = ["vsd.jpg", "fdsdb.jpg", "MV5BMjAzNjYwNzkxMl5BMl5BanBnXkFtZTgwMTU1OTk4MDE@._V1_UY317_CR16,0,214,317_AL_.jpg", "benjamin-parker-OhKElOkQ3R-unsplash-cropped.jpg"];
+var i = 0;
+var p = "images/";
+$(document).ready(function () {
+    $("#slide").attr("src", images[i]);
+    $("#left").click(function () {
+        $("#slide").attr("src", images[Math.abs(i % images.length)]);
```
Comment (brycebenson) [inline lesson04/assignment/index.js]: You'd need to add your fixed path variable to the `src` path for this to work.

File: lesson04/assignment/css/style.scss [hunk 3]
```diff
@@ -1,11 +1,30 @@
-#one{
-  height: 400px;
-  width: 400px;
-  background-color: blue;
+$bg: whitesmoke;
+@mixin b_default{
```
Comment (brycebenson) [inline lesson04/assignment/css/style.scss]: Typically a mixin would take in some value passed it to use, but this works.

File: lesson04/assignment/index.js (hunk 1 above)
Comment (awsirkis) [reply lesson04/assignment/index.js]: fixed

File: lesson04/assignment/index.js (hunk 2 above)
Comment (awsirkis) [reply lesson04/assignment/index.js]: fixed by adding path to the images array

**Pull request merged**